src/lib/Core.jsx

```diff
@@ -166,7 +166,7 @@ const Core = function ({
 
       return (
         <div className="result-answer-wrapper" key={index + 1}>
-          <h3 dangerouslySetInnerHTML={rawMarkup(`Q${question.questionIndex}: ${question.question}`)} />
+          <h3 dangerouslySetInnerHTML={rawMarkup(`Q${question.questionIndex}: ${question.question}  ${appLocale.marksOfQuestion.replace('<marks>', question.point)}`)} />
```

Comment:
there is 2 spaces before `${appLocale.marksOfQuestion.replace('<marks>', question.point)}`)}`. Please delete one space.